tests/oclinter/identity-case/openconfig-testcase-fail.yang

```diff
@@ -0,0 +1,25 @@
+module openconfig-testcase-fail {
+  prefix "oc-tc";
+  namespace "http://openconfig.net/linter/testcase";
+
+  import openconfig-extensions { prefix oc-ext; }
+
+  description
+    "Simple testcase to check compliance with quoting
+    of strings";
+
+  oc-ext:openconfig-version "0.0.1";
+
+  revision 2016-09-28 {
+    reference "0.0.1";
+    description
+      "Revision statement";
+  }
+
+  grouping testcase-top {
+    description
+      "Placeholder";
+  }
+
+  identity lowercase;
```

Comment:
for both enums and identities, do we need tests for some of other cases we've discovered (e.g., hex values with mix lower/upper case) ?